packages/seller-client/src/components/TotalBreakdown/TotalBreakdown.tsx

```diff
@@ -0,0 +1,47 @@
+import type { BreakdownValue } from './types';
+
+import React from 'react';
+import { Box, Typography } from '@material-ui/core';
+
+type Props = {
+  values: BreakdownValue[];
+  totalLabel?: string;
+};
+
+const TotalBreakdown: React.FC<Props> = ({ values, totalLabel }) => {
+  const total = values.reduce(
+    (sum, currentValue) => sum + currentValue.amount,
+    0,
+  );
+  return (
+    <Box width="100%">
+      {values.map((value, index) => (
+        <Box
+          key={value.label}
+          display="flex"
+          flexDirection="row"
+          justifyContent="space-between"
+          marginTop={index !== 0 && 0.5}
+        >
+          <Typography variant="body2">{value.label}</Typography>
+          <Typography variant="body2">
+            {`${value.amount < 0 ? '-' : ''}$${Math.abs(value.amount)}`}
+          </Typography>
+        </Box>
+      ))}
+      <Box
+        display="flex"
+        flexDirection="row"
+        justifyContent="space-between"
+        marginTop={0.5}
+      >
+        <Typography variant="body1">{totalLabel}</Typography>
```

Comment:
We should have a default total label
```suggestion
        <Typography variant="body1">{totalLabel ?? DEFAULT_TOTAL_LABEL}</Typography>
```

Reply:
Ah thanks for the catch. Updated in 8711b755f349b875bfee86026614513dda04d855